task.sql

```diff
@@ -9,9 +9,32 @@
 ) ENGINE=InnoDB;
 
 -- Create a table for caching GeoIP data (Columns: ID, IP Range, CountryID)
+CREATE TABLE GeoIP (
+    ID INT AUTO_INCREMENT PRIMARY KEY,
+    IPRange VARCHAR (50),
+    CountryID INT
+) ENGINE=MEMORY;
```

Comment:
The `GeoIP` table uses the `MEMORY` storage engine, which is suitable for temporary data but not for persistent storage. If you need to store data persistently, consider using a different storage engine like `InnoDB`.